ci-operator/step-registry/gcp/validate/bastionhost/gcp-validate-bastionhost-commands.sh

```
@@ -0,0 +1,520 @@
+    [[ $- == *x* ]] && WAS_TRACING=true || WAS_TRACING=false
+    set +x
+
+    # Try multiple SSH approaches to handle different OpenSSH versions and key formats
+    local base_options="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=30 -o ServerAliveCountMax=5 -o ConnectTimeout=10"
+    local ret=1
+    local attempt=0
+
+    # Approach 1: Standard modern SSH with primary key (supports OpenSSH format natively)
+    ((attempt+=1))
+    log "DEBUG: SSH Attempt ${attempt}: Standard SSH with primary key"
+    output=$(ssh ${base_options} -i "${sshkey}" ${user}@${host} "${remote_cmd}" 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)